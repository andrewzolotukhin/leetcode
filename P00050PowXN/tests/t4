1.0000001 2147483647
