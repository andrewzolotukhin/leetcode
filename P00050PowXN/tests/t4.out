1.83664e+93
